README.md

```diff
@@ -23,17 +22,17 @@ Variable                          | Default
 --------------------------------- | -------------------------------------------------------------------- | -----------------------------------
 MONGODB_HOST                      | chs-mongo (in Vagrant) / localhost                                   | Mongo database host.
 MONGODB_PORT                      | Environment specific port / 27017                                    | Mongo database port.
-ITEMS_DATABASE                    | items                                                                | MongoDB Items database name.
-MONGODB_URL                       | $MONGODB_URL/$ITEMS_DATABASE                                         | Mongo database URL.
-ITEMS_ORDERS_API_CH_GOV_UK_PORT   | 10020 (in Vagrant)                                                   | API port.
-ITEMS_ORDERS_API_CH_GOV_UK_URL    | http://${API_DOMAIN}:${ITEMS_ORDERS_API_CH_GOV_UK_PORT}              | API URL.
+CERTIFICATES_DATABASE             | certificates                                                         | MongoDB Certificates database name.
```

Comment:
Are we changing the database name? I couldn't see that in the story checklist.

Reply:
This has now been changed back to `items`